extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/KafkaTuningConfig.java

```diff
@@ -273,6 +310,8 @@ public String toString()
            ", reportParseExceptions=" + reportParseExceptions +
            ", handoffConditionTimeout=" + handoffConditionTimeout +
            ", resetOffsetAutomatically=" + resetOffsetAutomatically +
+           ", sinkFactory=" + sinkFactory +
+           ", customIndexMerger=" + customIndexMerger +
```

Comment:
IndexMerger is interface now, the implement of this interface should implement toString